### ISSUE TITLE
Grizzly G0750G Implementation

### ISSUE THREAD

**Issue by K5FR** [clough42/electronic-leadscrew#184]
I just finished installing the ELS on my G0750G lathe and am totally happy with the result. My biggest concern when starting this mod was finding a stepper/servo that would handle the load of this lathe. I settled on a ClearPath CPM-SDSK-3446P-RLN servo which is probably an over-design, but I didn't want to have to try the lesser motors and spend a lot of time swapping them out for bigger ones. Mounting the motor was about the same as James did his using the gear spider with the exception I used a 5mm HTD belt and pulleys. Everything else was just follow the instructions James provided. I also added a stop switch for turning to a shoulder and it works well.

**Comment by pmelevende** [comment]
Yes the videos, instructions, and answers to peoples questions make this project a standout. I also completed mine recently and couldn't be happier.
Thanks
Pauly

**Comment by koenbro** [comment]
> I just finished installing the ELS on my G0750G lathe and am totally happy with the result. My biggest concern when starting this mod was finding a stepper/servo that would handle the load of this lathe. I settled on a ClearPath CPM-SDSK-3446P-RLN servo which is probably an over-design, but I didn't want to have to try the lesser motors and spend a lot of time swapping them out for bigger ones. Mounting the motor was about the same as James did his using the gear spider with the exception I used a 5mm HTD belt and pulleys. Everything else was just follow the instructions James provided. I also added a stop switch for turning to a shoulder and it works well.

Can you please tell me how you wired the Clearpath servo? Thank you kindly.

**Comment by K5FR** [comment]
See this entry in my Toys blog for how I did the basic installation.
https://www.k5fr.com/Blog/?p=338

**Comment by koenbro** [comment]
Very helpful thank you. I managed to get it turning.
